students/18th/18기 양지은/login.js

```diff
@@ -0,0 +1,20 @@
+
+document.addEventListener('keyup', function() {
+    const userId = document.getElementById('userid');
+    const userPassword = document.getElementById('password');
+
+    if (userId.value) {
+        if (userPassword.value) {                        
+            let color = document.getElementsByClassName('btn')[0];
+            color.style.backgroundColor = "#1565c0";
+        }
+    }
+    if (!userId.value) {
+            let color = document.getElementsByClassName('btn')[0];
+            color.style.backgroundColor = "lightskyblue";
+    }
+    if (!userPassword.value) {
+        let color = document.getElementsByClassName('btn')[0];
+        color.style.backgroundColor = "lightskyblue";
+    }
```

Comment:
하나의 요소를 해당하는 조건에 따라 다른 실행문이 실행되는 구조이므로 아래와 같이 작성하여 반복되는 코드를 줄일 수 있습니다.
```suggestion
    const btn = document.getElementsByClassName('btn')[0];
    const condition = userId.value && userPassword.value;
    btn.style.backgroundColor = condition ? "#1565c0" : "lightskyblue";
```

Reply:
넵 참고하겠습니다!